folio-spring-i18n/src/main/java/org/folio/spring/config/TranslationConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.spring.config;
+
+import java.util.Locale;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * An {@link org.springframework.context.annotation.Configuration Configuration} bean to hold the
+ * location of the current translation directory.  By default, this is set from the application
+ * property {@code folio.translation=directory} in your {@code application.properties} or
+ * {@code application.yaml} file.  Note that the proceeding and trailing slashes are required.
+ */
+@Getter
+@Configuration
+public class TranslationConfiguration {
+
+  private String translationDirectory;
+
+  private Locale defaultLocale;
+
+  @Autowired
+  public TranslationConfiguration(
+    @Value(
+      "${folio.translation-directory:/translations/}"
```

Comment:
@mweaver-ebsco idk why I didn't see your reply earlier; it's relative to resources (as we can't reliably access paths outside of there, particularly once it's packaged).  I suppose this is redundant, though, since `/translations` already has to be specified in the POM:
```
    <resource>
      <directory>${project.basedir}/translations</directory>
      <targetPath>translations</targetPath>
    </resource>
```

Since it's already there, the `directory` could be whatever. I'll update this to remove the parameter.

The only other use case I could think of if they were centralized (as they are for the data import modules in https://github.com/folio-org/data-import-raml-storage, for example), but the above can handle that issue.